cmd/juju/caas/add.go

```diff
@@ -409,10 +429,7 @@ func (c *AddCAASCommand) Run(ctx *cmd.Context) error {
 	}
 
 	if clusterName == "" {
-		clusterName = c.hostCloudRegion
-		if clusterName == "" {
-			clusterName = newCloud.HostCloudRegion
-		}
+		clusterName = newCloud.HostCloudRegion
```

Comment:
Not needed?

Reply:
yeah, we should just use the validated/ensured newCloud.HostCloudRegion without trying the c.hostCloudRegion